### DISCUSSION TITLE
Scene 이동 방법

### DISCUSSION THREAD

**Discussion by HyunBin3** [naverz/zepeto-studio-kor#1189] [5. Q&A]
오브젝트랑 플레이어가 충돌이 일어나면 씬을 이동시키고 싶은데 뜻대로 되지 않네요.. 뭐가 문제인지도 잘 모르겠구요

import { Collider } from 'UnityEngine';
import { SceneManager } from 'UnityEngine.SceneManagement';
import { ZepetoCharacter, ZepetoPlayers } from 'ZEPETO.Character.Controller';
import { ZepetoScriptBehaviour } from 'ZEPETO.Script'

export default class Item extends ZepetoScriptBehaviour {

    private zepetoCharacter: ZepetoCharacter;
    public SceneManager: SceneManager;

    Start() {
        ZepetoPlayers.instance.OnAddedLocalPlayer.AddListener(() => {
            this.zepetoCharacter = ZepetoPlayers.instance.LocalPlayer.zepetoPlayer.character;
        });
    }

    OnTriggerEnter(collider: Collider) {

        if (this.zepetoCharacter == null || collider.gameObject != this.zepetoCharacter.gameObject)
            return;

        if (collider.gameObject.name == "Pig") {
            SceneManager.LoadScene(SampleScene);
            SceneManager.LoadScene("SampleScene", LoadSceneMode.Single);
            
        }

        this.gameObject.SetActive(false);

    }

    

}

**Comment by YSHFM** [comment]
에러메시지가 어떻게 나오는 지 알 수 있을까요??
